packages/ui/feature-dashboard/src/lib/pages/flows-table/flows-table.component.html

```diff
@@ -35,14 +35,18 @@
             <ng-container matColumnDef="status">
               <th mat-header-cell *matHeaderCellDef>Status</th>
               <td mat-cell *matCellDef="let flow">
-                <div class="ap-flex ap-gap-2">
+                <div class="ap-flex ap-gap-2 ap-items-center ap-justify-center">
                   <mat-slide-toggle color="primary" [class.ap-cursor-not-allowed]="flow.instanceToggleControl.disabled"
                     (click)="$event.stopImmediatePropagation(); toggleFlowStatus(flow,flow.instanceToggleControl)"
                     [formControl]="flow.instanceToggleControl" [matTooltip]="flow.instanceToggleControl.disabled? 'Please publish the flow':
                   flow.instanceToggleControl.value? 'Flow is on' : 'Flow is off'"></mat-slide-toggle>
-                  <div [class.ap-invisible]=" !flowsUpdateStatusRequest$[flow.id]" class="spinner-border dark-spinner"
-                    role="status">
-                  </div>
```

Comment:
Why did you remove this loader? it is meant to show up when someone switches off/on the instance

Reply:
Can you handle this?